raiden/network/transport/matrix/rtc/web_rtc.py

```diff
@@ -92,6 +103,23 @@ def spawn_create_channel(self, partner_address: Address) -> None:
         )
         self._spawn_web_rtc_coroutine(coroutine, self._handle_sdp_callback, partner_address)
 
+    def set_candidates(
+        self, partner_address: Address, description: Dict[str, Union[str, Sequence[str]]]
+    ) -> None:
+        assert self.node_address, "Transport is not started yet but tried to set candidates"
+        rtc_partner = self.get_rtc_partner(partner_address)
+        msg = "Partner RTCPeerConnection is not set, but tried to set candidates"
+        assert rtc_partner.peer_connection is not None, msg
+        msg = "Partner remote description is not set, but tried to set candidates"
+        assert rtc_partner.peer_connection.remoteDescription is not None, msg
```

Comment:
have you checked if it is possible to set candidates before setting remote description?